docs/source/public_api/filtsmooth.rst

```diff
@@ -19,8 +19,9 @@ probnum.filtsmooth.statespace
 
 Probabilistic State Space Models.
 
-This package implements continuous-discrete and discrete-discrete state space models,
-which are the basis for Bayesian filtering and smoothing, but also probabilistic ODE solvers.
+This package implements continuous-discrete and discrete-discrete state
+space models, which are the basis for Bayesian filtering and smoothing,
+but also probabilistic ODE solvers.
```

Comment:
do we have different automatic formatting tools? I remember that changes like these crop up a few times and if it is something that can be resolved for good, it would declutter PRs.

Reply:
I think this occurred since I played around with how we design the documentation (read from the docstrings or text in restructuredText files). If we both use docformatter via the pre-commit hook, there shouldn't be any differences. Alternatively we could add docformatter to Github Actions as well via tox.